perses/app/relative_point_mutation_setup.py

```diff
@@ -23,6 +23,10 @@
 beta = 1.0/kT
 ring_amino_acids = ['TYR', 'PHE', 'TRP', 'PRO', 'HIS']
 
+# Set up logger
+_logger = logging.getLogger()
+_logger.setLevel(logging.DEBUG)
```

Comment:
I personally think this should be INFO everywhere but it's something I fight with Dom with a lot